app/views/user_mailer/mail_new_event.html.erb

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+      /* Email styles need to be inline */
+    </style>
+  </head>
+
+  <body>
+    <p>Hallo <%= @user.name %>,</p>
+    <p>Er is een nieuwe activiteit genaamd '<%= @event.title %>' aangemaakt.</p>
+    <p>Schrijf je zo snel mogelijk in dan wel uit op https://www.zondersikkel.nl/events/<%= @event.id %></p>
```

Comment:
Kan dit een `<a href="mailto:x">y</a>` worden?

Reply:
Uhh snap niet helemaal wat je hier bedoelt. Deze is voor in de mail zodat mensen snel naar de activiteit kunnen klikken, niet om een mail te sturen

Reply:
Oh ja mailto was niet handig, dat moest er niet in. Maar een linkje is misschien wel nice:

`Schrijf je zo snel mogelijk in dan wel uit op <a href="https://zondersikkel.nl/etc/etc">onze website</a>!`

Reply:
Als het goed is doet hij dat automatisch, zo staat ie ook in views/user_mailer/mail_event_reminder en die maakt er ook automatisch een link van.

Reply:
Oh okay top. Zo niet dan zie je het vanzelf als je gaat testen haha

Reply:
```suggestion
    <p>Schrijf je zo snel mogelijk in dan wel uit op <%= event_url(event, host:" https://zondersikkel.nl")
</p>
```

Waarschijnlijk is host al gezet zelfs op productie en kan je die weglaten maar dat moet @dexbleeker je vertellen :)